gcsio/src/main/java/com/google/cloud/hadoop/gcsio/testing/FakeServer.java

```diff
@@ -0,0 +1,45 @@
+package com.google.cloud.hadoop.gcsio.testing;
+
+import com.google.cloud.NoCredentials;
+import com.google.cloud.storage.GrpcStorageOptions;
+import com.google.cloud.storage.StorageOptions;
+import com.google.storage.v2.StorageGrpc;
+import io.grpc.Server;
+import io.grpc.netty.shaded.io.grpc.netty.NettyServerBuilder;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.util.concurrent.TimeUnit;
+
+public final class FakeServer implements AutoCloseable {
```

Comment:
This has to be public?

Reply:
Yes, This is part of testing package which wont be visible to tests otherwise.